src/main/scala/com/microsoft/ml/spark/vw/VowpalWabbitBase.scala

```diff
@@ -163,6 +166,20 @@ trait VowpalWabbitBase extends Wrappable
   def getNumBits: Int = $(numBits)
   def setNumBits(value: Int): this.type = set(numBits, value)
 
+  val exportReadableModel = new BooleanParam(this, "exportReadableModel",
+    "If true exports the readable model")
+
+  def getExportReadableModel: Boolean = $(exportReadableModel)
```

Comment:
Is there a way to make this a method that performs the action as opposed to a flag that decides whether to perform the action?

Reply:
before training it must be known that the readable model needs to be accessed afterwards (triggers additional computation)